### PR TITLE
Add chartAttributes support

### PR DESCRIPTION
**Pull request by bgreg1012**
I tested on our internal wavefront servers and it looks good. Please lmk if you need any updates.

### CODE REVIEW

File: dashboard.go [hunk 1]
```diff
@@ -141,6 +141,9 @@ type Chart struct {
 
 	// ChartSettings are custom settings for the chart
 	ChartSettings ChartSetting `json:"chartSettings"`
+
+	// ChartSettings are custom settings for the chart
```
Comment (louism517) [inline dashboard.go]: Could you just change this to reflect ChartAttributes?

**Review by louism517 — CHANGES_REQUESTED** [review]
Thanks for your PR. Looks good, just one small comment to be changed and then happy to merge and release.

File: dashboard.go (hunk 1 above)
Comment (bgreg1012) [reply dashboard.go]: Oops, yeah will do.

Comment (bgreg1012) [reply dashboard.go]: done

**Pull request merged**